spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/web/servlet/ServletManagementChildContextConfiguration.java

```diff
@@ -147,12 +149,12 @@ abstract static class AccessLogCustomizer implements Ordered {
 
 		private static final String MANAGEMENT_PREFIX = "management_";
 
-		protected String customizePrefix(String prefix) {
+		protected String customizePrefix(String prefix, String accessLogPrefix) {
 			prefix = (prefix != null) ? prefix : "";
 			if (prefix.startsWith(MANAGEMENT_PREFIX)) {
```

Comment:
This looks wrong to me. It should check against `accessLogPrefix`, not against a hardcoded "management_".

Reply:
Was note really sure about doing this, but it can be changed easily